Cargo.toml

```
@@ -12,14 +12,20 @@ repository = "https://github.com/datafuselabs/opendal"
 version = "0.11.3"
 
 [package.metadata.docs.rs]
-all-features = true
+features = ["compres", "retry", "serde", "services-hdfs", "services-http"]
 
 [features]
+default = ["epoll"]
 compress = ["async-compression"]
 retry = ["backon"]
 serde = ["time/serde"]
 services-hdfs = ["hdrs"]
 services-http = ["radix_trie"]
+# the io features, by default, we use `epoll`
+# to enable `iouring` we need to disable the `default` feature
+epoll = ["nuclei/epoll"]
```

```suggestion
__epoll = ["nuclei/epoll"]
```

Renaming as `__epoll` to indicate this is an internal feature we don't want users to use. And `epoll` should only be used on the `linux` platform. Maybe we should hide it under the cfg?

---

epoll must be enable on linux , emmm, I don't have any other equipment at the moment and am not quite sure if it will affect other platforms. Can you help?

---

Use the default feature of `nuclei` LGTM.